src/core/zowe/core_for_zowe_sdk/logger.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import os
+
+class Log:
+    """root logger setup and a funtion to customize logger level"""
+
+    dirname = os.path.join(os.path.expanduser("~"), ".zowe/logs")
+
+    if not os.path.isdir(dirname):
+        os.makedirs(dirname)
```

Comment:
This could be optimized to just one line:
```suggestion
    os.makedirs(dirname, exist_ok=True)
```